config.go

```diff
@@ -32,7 +32,7 @@ var (
 	BuildRevision = ""
 
 	// Commandline Version String; used to control auto-updating.
-	ClientVersion = "2021-12-17"
+	ClientVersion = "2021-12-22"
```

Comment:
Need to increment one more time, unfortunately its updated again since we started review.